docs/performance-scoring.md

```diff
@@ -0,0 +1,40 @@
+# Goal
+
+The goal of this document is to explain how performance scoring on Lighthouse works and what to do to improve your Lighthouse performance score. 
+
+# What performance metrics does Lighthouse measure?
+
+Lighthouse measures the following performance metrics: 
+
+- [First meaningful paint](https://developers.google.com/web/tools/lighthouse/audits/first-meaningful-paint): first meaningful paint is defined as when the browser first puts any “meaningful” element/set of “meaningful” elements on the screen. What is meaningful is determined from a series of heuristics. 
+- [First interactive](https://developers.google.com/web/tools/lighthouse/audits/first-interactive): first interactive is defined as the first point at which the page could respond quickly to input. It doesn't consider any point in time before first meaningful paint. The way this is implemented is primarily based on heuristics. 
+*Note: this metric is currently in beta, which means that the underlying definition of this metric is in progress.*
+- [Consistently interactive](https://developers.google.com/web/tools/lighthouse/audits/consistently-interactive): defined as the first point at which everything is loaded such that the page will quickly respond to any user input throughout the page. 
+*Note: this metric is currently in beta, which means that the underlying definition of this metric is in progress.*
+- [Perceptual Speed Index (pSI)](https://developers.google.com/web/tools/lighthouse/audits/speed-index): pSI measures how many pixels are painted at each given time interval on the viewport. The fewer the pixels that are painted, the better since we want an experience where most of the content is shown on the screen during the first few moments of initiating the page load. Therefore, the lower the pSI score, the better. 
+- [Estimated Input Latency](https://developers.google.com/web/tools/lighthouse/audits/estimated-input-latency):  this audit measures how fast your app is in responding to user input. Our benchmark is that the estimated input latency should be under 50 ms (see documentation here as to why).
+
+*Note that Lighthouse can experience weird behaviors when it runs in the presence of anti-virus scanners and other extensions or programs that interfere with page load. Please try to run without anti-virus scanners or other extensions/programs to get the cleanest results.*
```

Comment:
little tidbit I think might help head off metric instability concerns too, but feel free to edit/disagree

"Important to keep in mind that these are user-focused metrics attempting to capture how the user experiences page load. If a user wouldn't have the exact same experience across different visits, then Lighthouse won't report them being the same either. Some variability when running on real-world sites is to be expected as sites load different ads, scripts, and network conditions vary for each visit."

Reply:
Added some wording.

Reply:
Added some stuff to address your concerns.